include/nbl/asset/IBuffer.h

```diff
@@ -19,31 +19,48 @@ class IBuffer : public core::IBuffer, public IDescriptor
 		E_CATEGORY getTypeCategory() const override {return EC_BUFFER;}
 
 		//!
-        enum E_USAGE_FLAGS : uint32_t
-        {
-            EUF_NONE = 0x00000000,
-            EUF_TRANSFER_SRC_BIT = 0x00000001,
-            EUF_TRANSFER_DST_BIT = 0x00000002,
-            EUF_UNIFORM_TEXEL_BUFFER_BIT = 0x00000004,
-            EUF_STORAGE_TEXEL_BUFFER_BIT = 0x00000008,
-            EUF_UNIFORM_BUFFER_BIT = 0x00000010,
-            EUF_STORAGE_BUFFER_BIT = 0x00000020,
-            EUF_INDEX_BUFFER_BIT = 0x00000040,
-            EUF_VERTEX_BUFFER_BIT = 0x00000080,
-            EUF_INDIRECT_BUFFER_BIT = 0x00000100,
-            EUF_SHADER_DEVICE_ADDRESS_BIT = 0x00020000,
-            // we will not expose transform feedback
+		enum E_USAGE_FLAGS : uint32_t
+		{
+			EUF_NONE = 0x00000000,
+			EUF_TRANSFER_SRC_BIT = 0x00000001,
+			EUF_TRANSFER_DST_BIT = 0x00000002,
+			EUF_UNIFORM_TEXEL_BUFFER_BIT = 0x00000004,
+			EUF_STORAGE_TEXEL_BUFFER_BIT = 0x00000008,
+			EUF_UNIFORM_BUFFER_BIT = 0x00000010,
+			EUF_STORAGE_BUFFER_BIT = 0x00000020,
+			EUF_INDEX_BUFFER_BIT = 0x00000040,
+			EUF_VERTEX_BUFFER_BIT = 0x00000080,
+			EUF_INDIRECT_BUFFER_BIT = 0x00000100,
+			EUF_SHADER_DEVICE_ADDRESS_BIT = 0x00020000,
+			// we will not expose transform feedback
 			//EUF_TRANSFORM_FEEDBACK_BUFFER_BIT_EXT = 0x00000800,
-            //EUF_TRANSFORM_FEEDBACK_COUNTER_BUFFER_BIT_EXT = 0x00001000,
-            EUF_CONDITIONAL_RENDERING_BIT_EXT = 0x00000200,
-            EUF_ACCELERATION_STRUCTURE_BUILD_INPUT_READ_ONLY_BIT = 0x00080000,
-            EUF_ACCELERATION_STRUCTURE_STORAGE_BIT = 0x00100000,
-            EUF_SHADER_BINDING_TABLE_BIT = 0x00000400,
+			//EUF_TRANSFORM_FEEDBACK_COUNTER_BUFFER_BIT_EXT = 0x00001000,
+			EUF_CONDITIONAL_RENDERING_BIT_EXT = 0x00000200,
+			EUF_ACCELERATION_STRUCTURE_BUILD_INPUT_READ_ONLY_BIT = 0x00080000,
+			EUF_ACCELERATION_STRUCTURE_STORAGE_BIT = 0x00100000,
+			EUF_SHADER_BINDING_TABLE_BIT = 0x00000400,
 			//! synthetic Nabla inventions
 			// whether `IGPUCommandBuffer::updateBuffer` can be used on this buffer
 			EUF_INLINE_UPDATE_VIA_CMDBUF = 0x80000000u,
+
+			EUF_ANY = EUF_TRANSFER_SRC_BIT
```

Comment:
rename to ALL, however you'll get huge validation and creation problems if you actually try to use it, because if you don't enable or have acceleration structure/raytracing them it will fail to create the buffer

What do you need it for?